spec/services/remote_identities/create_service_spec.rb

```diff
@@ -4,23 +4,31 @@
 
 require "services/base_services/behaves_like_create_service"
 
-RSpec.describe RemoteIdentities::CreateService, type: :model do
+RSpec.describe RemoteIdentities::CreateService, :webmock, type: :model do
   let(:user) { create(:user) }
   let(:storage) { create(:nextcloud_storage_configured) }
   let(:oauth_config) { storage.oauth_configuration }
-  let(:oauth_token) { Rack::OAuth2::AccessToken.new(access_token: "sudo-access-token", user_id: "bob-from-accounting") }
+  let(:oauth_client_token) do
+    create(:oauth_client_token,
+           user:,
+           oauth_client: oauth_config.oauth_client)
+  end
+
+  subject(:service) { described_class.new(user:, oauth_config:, oauth_client_token:) }
 
-  subject(:service) { described_class.new(user:, oauth_config:, oauth_token:) }
+  before do
+    stub_request(:get, File.join(storage.host, "/ocs/v1.php/cloud/user")).to_return(status: 200, body: File.read("modules/storages/spec/support/payloads/nextcloud_user_query_success.xml"), headers: {})
```

Comment:
Seeing that there are quite a bunch of line length offenses for essentially the same stubbed line. What do you think about offering a centralized helper to mock this?

E.g. `stub_nextcloud_user_query(storage.host)`

Reply:
Yes, good idea. Will do.